### PR TITLE
Valid second-level + TLD combinations should not get recommendations

### PR DESCRIPTION
**Pull request by hjwp**
cf #97, not sure if y'all actually want this but thought I'd put it out there in case you did.

### CODE REVIEW

**Comment by weilu** [comment]
LGTM. Will merge after a 2nd ACK. @kevinhughes27 @derrickko ?

File: spec/mailcheckSpec.js [hunk 1]
```diff
@@ -138,6 +138,16 @@ describe("mailcheck", function() {
...
+      it("will not offer suggestions for valid 2ld-tld combinations", function() {
+        expect(
+            mailcheck.suggest('test@gmail.co.uk', domains, secondLevelDomains, topLevelDomains)
+        ).toBeFalsy();
+
+        expect(
+            mailcheck.suggest('test@gmail.fr', domains, secondLevelDomains, topLevelDomains)
+        ).toBeFalsy();
+      });
     });
```
Comment (kevinhughes27) [inline spec/mailcheckSpec.js]: can you use hotmail or another second level domain for this test please - the idea behind `domains` is that they only have one valid tld and shoudn't be compared so having gmail in both lists is a missleading test case imo

Comment (derrickko) [inline spec/mailcheckSpec.js]: +1 to that.

**Comment by hjwp** [comment]
I've used the "gmx" domain which is unused elsewhere, and split it out into two tests, one of which is explicitly testing the fact that you don't get a recommendation even if there *is* a fully-specified domain that matches...

**Comment by kevinhughes27** [comment]
lgtm

**Pull request merged**